x-pack/plugins/apm/server/routes/alerts/rule_types/error_count/register_error_count_rule_type.ts

```diff
@@ -217,8 +208,22 @@ export function registerErrorCountRuleType({
                 interval: `${ruleParams.windowSize}${ruleParams.windowUnit}`,
                 reason: alertReason,
                 serviceName,
-                threshold: ruleParams.threshold,
-                triggerValue: errorCount,
+                threshold: i18n.translate(
```

Comment:
This is a breaking change no?

Reply:
Why?

Reply:
People use these variables in templates. This will change what message is generated from that template.

Reply:
ah, you mean `{{context.threshold}}`, right?!

Reply:
Yes

Reply:
yeah, you're right, I'll revert it.